### PR TITLE
Tl/functions

### PR DESCRIPTION
**Pull request by Torpheus**

### CODE REVIEW

File: src/cache.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#ifndef CACHE_GLOB
+#define CACHE_GLOB
+
+#include <stdlib.h>
```
Comment (rapgru) [inline src/cache.h]: remove unused includes

File: src/cache/cache.h [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#ifndef CACHE
+#define CACHE
+
+#include <stddef.h>
```
Comment (rapgru) [inline src/cache/cache.h:4]: remove include

File: src/cache/cache.c [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#include "cache.h"
+#include "../cache.h"
+
```
Comment (rapgru) [inline src/cache/cache.c:5]: check if include stdref.h is neccessary

File: src/functions/functions.c [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#include "functions.h"
+#include "../functions.h"
+
+activation_t* current_activation;
+activation_t* temporary_activation;
+Dyntype return_value;
```
Comment (rapgru) [inline src/functions/functions.c]: dyntype_t instead of Dyntype

**Review by rapgru — CHANGES_REQUESTED** [review]

File: src/functions/functions.c (hunk 4 above)
Comment (Torpheus) [reply src/functions/functions.c]: will change in merge commit

File: src/cache/cache.h (hunk 2 above)
Comment (rapgru) [reply src/cache/cache.h:4]: `size_t` is used in header file

**Review by rapgru — DISMISSED** [review]

**Review by rapgru — DISMISSED** [review]

**Review by rapgru — APPROVED** [review]

**Pull request merged**